src/app/Plans/components/MigrateOrCutoverConfirmModal.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import ConfirmModal, { IConfirmModalProps } from '@app/common/components/ConfirmModal';
+import { IPlan } from '@app/queries/types';
+import { TextContent, Text } from '@patternfly/react-core';
+
+interface IMigrateOrCutoverConfirmModalProps
+  extends Pick<IConfirmModalProps, 'isOpen' | 'toggleOpen' | 'mutateFn' | 'mutateResult'> {
+  plan: IPlan;
+  action: 'start' | 'restart' | 'cutover';
+}
+
+const MigrateOrCutoverConfirmModal: React.FunctionComponent<IMigrateOrCutoverConfirmModalProps> = ({
+  isOpen,
+  toggleOpen,
+  mutateFn,
+  mutateResult,
+  plan,
+  action,
+}: IMigrateOrCutoverConfirmModalProps) => {
+  const verb = action === 'restart' ? 'Restart' : 'Start';
+  const noun =
+    action === 'cutover' ? 'cutover' : plan.spec.warm ? 'incremental copies' : 'migration';
```

Comment:
@vconzola 
I have a question about "incremental copies". Isn't this called "stage migration" in other parts of the UI? I think that is how the Migration Plan wizard refers to the step before cutover and that is the term I have used in the documentation.